go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	golang.org/x/crypto v0.23.0
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.29.4
-	k8s.io/apimachinery v0.29.4
+	k8s.io/apimachinery v0.30.1
```

Comment:
Quick this does match release/v2.9 version?

Reply:
No it doesn't look like they match.

Reply:
We don't need this version bump then right?

Reply:
I don't believe so but i'll look into it.

Reply:
Fixed this to not include the version bump.